jaeger-core/src/main/java/io/jaegertracing/senders/HttpSender.java

```diff
@@ -31,6 +32,7 @@
 import okhttp3.Response;
 
 @ToString(exclude = {"httpClient", "requestBuilder"})
+@Slf4j
 public class HttpSender extends ThriftSender {
```

Comment:
Looks like your branch was still using an old master. As part of #449, this has been moved to `jaeger-thrift/src/main/java/io/jaegertracing/thrift/senders/HttpSender.java`

Please, rebase this to the latest master.